core/txpool/legacypool/legacypool.go

```diff
@@ -746,6 +765,10 @@ func (pool *LegacyPool) add(tx *types.Transaction, local bool) (replaced bool, e
 		knownTxMeter.Mark(1)
 		return false, txpool.ErrAlreadyKnown
 	}
+
+	maxMainPoolSize := pool.config.GlobalSlots + pool.config.GlobalQueue
```

Comment:
line 769, 770, 809, 829,830
it has no logic change in this PR
I suggest to not change the raw code, for easily upstream in the future

for other empty lines added in this PR, I also suggest to roll back